modules/javafx.graphics/src/main/java/com/sun/marlin/DualPivotQuicksort20191112Ext.java

```diff
@@ -49,7 +49,7 @@
  */
 public final class DualPivotQuicksort20191112Ext {
 
-    private static final boolean FAST_ISORT = false;
+    private static final boolean FAST_ISORT = true;
```

Comment:
How much testing have you done with this flag set to true?

Reply:
This is a validated shortcut as dpqs main loop also uses insertionsort on small arrays too, functionally equivalent.